flowkit_jwt_generator/flowkit_jwt_generator/jwt.py

```diff
@@ -273,32 +237,39 @@ def print_token(username, secret_key, lifetime, audience):
 
     For example:
 
+
+    generate-jwt --username TEST_USER --private-key $PRIVATE_JWT_SIGNING_KEY --lifetime 1 --audience TEST_SERVER all-access http://localhost:9090
```

Comment:
This is missing the `--flowapi-url` option:
```suggestion
    generate-jwt --username TEST_USER --private-key $PRIVATE_JWT_SIGNING_KEY --lifetime 1 --audience TEST_SERVER all-access --flowapi-url http://localhost:9090
```

Reply:
Strangely, the `--flowapi-url` doesn't show up though when running `generate-jwt --help`?

Reply:
Ah yes. Little convoluted that - it'll only show up in the help for the subcommand (`all-access`). Annoyingly, `click` won't let you get to that bit unless you've successfully completed all the required named args, so you need to do something like `enerate-jwt --username TEST_USER --private-key $PRIVATE_JWT_SIGNING_KEY --lifetime 1 --audience TEST_SERVER all-access --help` to see it.